src/modules/TraceRouteModule.cpp

```diff
@@ -12,6 +12,17 @@ bool TraceRouteModule::handleReceivedProtobuf(const meshtastic_MeshPacket &mp, m
 void TraceRouteModule::alterReceivedProtobuf(meshtastic_MeshPacket &p, meshtastic_RouteDiscovery *r)
 {
     auto &incoming = p.decoded;
+    uint32_t now = millis();
+    if (incoming.request_id && lastTraceRouteBackMs && (now - lastTraceRouteBackMs) < (60 * 1000)) {
```

Comment:
`lastTraceRouteTowardMs` and `lastTraceRouteBackMs` don't seem to be set, but if you want to limit the reply generation, this should go only in `allocReply()`, because otherwise intermediate hops will not add themselves if another traceroute went through them already in the last minute.